packages/react-provider/src/Provider.tsx

```diff
@@ -0,0 +1,71 @@
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { getSlots, makeMergeProps } from '@fluentui/react-utils';
+import * as React from 'react';
+
+import { ThemeContext, ProviderContextValue, useFluent } from './context';
+import { PartialTheme } from '@fluentui/react-theme';
+import { ThemeProviderState, useThemeProviderState } from './ThemeProvider';
+
+export interface ProviderProps extends Partial<ProviderContextValue> {
```

Comment:
I'm not sure if we should have another `Provider`. We currently have 
1. `ThemeProvider` to provide `theme`. And `rtl` is defined inside `theme`.
2. `WindowProvider` to provider `document`

Reply:
1. I don't think that `rtl` setting is somehow related to theme. Will it change token values?
2. Are we going to create an another provider if we will propagate `telemetry` object, `performance` flags or `disableAnimations` setting? These things are a part of real use case:

https://github.com/microsoft/fluentui/blob/4f9fe37e9be56a7e2ceb81eec8dbe9f7aa6a64be/packages/fluentui/react-bindings/src/context.ts#L27-L34

Reply:
RE: 1 - agreed that `rtl` doesn't belong to `theme`
RE: 2 - fair enough :)